workspaces/ui-v2/src/optic-components/SupportLinks.ts

```diff
@@ -7,3 +7,8 @@ export const DocumentingYourApi = docsBaseLink + '/using/baseline';
 export const Contact = (subject: string) =>
   `mailto:aidan@useoptic.com?subject=${subject}`;
 export const GithubIssues = 'https://github.com/opticdev/optic/issues';
+
+export const RunOpticLink = docsBaseLink;
```

Comment:
@LouManglass are these right? I thought this is what you'd change?

Reply:
Updated in latest changeset.